tecken/libsentry.py

```diff
@@ -0,0 +1,400 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import logging
+from urllib.parse import urlparse, parse_qsl, urlencode
+import sys
+
+import markus
+import sentry_sdk
+from sentry_sdk.integrations.logging import ignore_logger
+
+
+SENTRY_MODULE_NAME = __name__
+metrics = markus.get_metrics(SENTRY_MODULE_NAME)
+logger = logging.getLogger(SENTRY_MODULE_NAME)
+
+
+MASK_TEXT = "[Scrubbed]"
+
+
+ALL_COOKIE_KEYS = object()
+ALL_QUERY_STRING_KEYS = object()
```

Comment:
This is a nice way to do constants that you can compare cheaply using Python `is` because there's only one of them and `is` does equality by memory address.